Doc/library/os.rst

```diff
@@ -3383,6 +3383,216 @@ features:
       Added the :attr:`st_birthtime` member on Windows.
 
 
+.. function:: statx(path, mask, flags=0, *, dir_fd=None, follow_symlinks=True)
+
+   Get the status of a file or file descriptor by performing a :c:func:`!statx`
+   system call on the given path.
+
+   *path* may be specified as either a string or bytes -- directly or
```

Comment:
nit: Could maybe just say "*path* shold be a :term:`path-like object` or an open file descriptor" (and leave the path-like glossary docs to talk about string/byes/Path).